locust/load_test_results_vit_gke.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>Let's add a <code>-q</code> flag to keep the outputs less verbose.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/deploy-hf-tf-vision-models/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>